src/NextGen/DonationForm/Blocks/DonationFormBlock/Block.php

```diff
@@ -103,33 +101,13 @@ private function createForm($attributes): Form
 
         $donationForm = new Form('DonationForm');
```

Comment:
@kjohnson does this form name need to be dynamic?

Reply:
@jonwaldstein something to consider. We have the form ID as a `uid`, which might work.

Reply:
[e19fb6b](https://github.com/impress-org/givewp-next-gen/pull/6/commits/e19fb6becc2b44945663d67c2b184dda74c1df1b)